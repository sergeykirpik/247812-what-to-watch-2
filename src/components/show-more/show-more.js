import React from 'react';
import PropTypes from 'prop-types';

const ShowMore = ({onClick}) => (
  <div className="catalog__more">
    <button onClick={onClick} className="catalog__button" type="button">Show more</button>
  </div>
);

ShowMore.propTypes = {
  onClick: PropTypes.func
};

export default ShowMore;
