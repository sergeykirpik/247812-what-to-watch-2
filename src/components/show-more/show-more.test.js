import React from 'react';
import renderer from 'react-test-renderer';
import {ShowMore} from './show-more.jsx';

it(`renders correctly`, () => {
  const tree = renderer
    .create(<ShowMore />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
