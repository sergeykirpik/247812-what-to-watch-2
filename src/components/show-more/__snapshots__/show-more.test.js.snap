// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="catalog__more"
>
  <button
    className="catalog__button"
    type="button"
  >
    Show more
  </button>
</div>
`;
